### ISSUE TITLE
Multiple server impls may be needed

### ISSUE THREAD

**Issue by paullewis** [GoogleChrome/application-shell#29]
Mainly PHP. My assumption (and I think it'd be good to have data around this) is that many people are on shared architectures where PHP is the predominant env.

**Comment by Garbee** [comment]
Any existing ideas on how this would be structured in the repository? I would love to help here if I could.

**Comment by addyosmani** [comment]
We're documenting the server-side architecture as part of #12 and should have a version nailed down soon that can be re-implemented in PHP. I think ideally it would be awesome to start to see derivative implementations in diff lang flavours for folks that can't use Node for their backend.

**Comment by addyosmani** [comment]
So we've now published the architecture blog posts. Imo, it could be neat for us to encourage folks to fork this repo and try to replace the Express based backend with their own. We could then link out to those implementations. Only concern with including the source in this repo is it might make maintainance and bug tracking a little tricker.

**Comment by Garbee** [comment]
I agree with the maintenance part for the code itself. The other main problem is, things like  PHP have numerous :heavy_multiplication_x: numerous ways of handling things. So, if the repo becomes big in that world for  some reason, there will be many issues asking for different packages to be used. I think this repo is clean enough that developers should be able to look and refactor into Ruby or PHP on their own.

**Comment by gauntface** [comment]
Discussing this with some of the tech writers it sounds like we are intending to document (on a high level) what possible options exist for each language / server architecture. Doing this on WebFundamentals may have the added bonus of people contributing to list to improve it over time.

In the mean time I'm inclined to say close this bug, but happy if others disagree.
